docs/apache-airflow-providers-sqlite/connections/sqlite.rst

```diff
@@ -23,14 +23,22 @@ The SQLite connection type provides connection to a SQLite database.
 
 Configuring the Connection
 --------------------------
-Host (required)
-    The host to connect to.
+Host (optional)
+    The host to connect to. This can either be a file on disk or an in-memory database. If not set, an in-memory database is being used.
 
-Schema (optional)
-    Specify the schema name to be used in the database.
+Extra (optional)
+    Specify the extra parameters (as json dictionary) that can be used in the sqlite connection.
+    See `Recognized Query Parameters <https://www.sqlite.org/uri.html>`_ for all supported parameters.
 
-Login (required)
-    Specify the user name to connect.
+URI format example
+^^^^^^^^^^^^^^^^^^
 
-Password (required)
-    Specify the password to connect.
+If serializing with Airflow URI:
+
+.. code-block:: bash
+
+   export AIRFLOW_CONN_SQLITE_DEFAULT='sqlite://host/?mode=ro'
+
+When specifying the connection as an environment variable in Airflow versions prior to 2.3.0, you need to specify the connection using the URI format.
```

Comment:
What changed with 2.3? (I don't know what I was a user am meant to take from this paragraph)

Reply:
This is a copy&paste from the snowflake one.

Reply:
Delete it then? It doesn't make any sense to me

Reply:
JSON format added: https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#json-format-example

So better clarify or remove it

Reply:
I have clarified it. :)